pontoon/base/views.py

```diff
@@ -79,6 +81,12 @@ def translate(request, locale, slug, part):
         'part': part,
         'project': project,
         'projects': projects,
+        'is_google_translate_supported':
+            True if settings.GOOGLE_TRANSLATE_API_KEY
+            else False,
+        'is_microsoft_translator_supported':
+            True if settings.MICROSOFT_TRANSLATOR_API_KEY
+            else False,
```

Comment:
Both of these can be simplified to `bool(settings.XXX_SERVICE_API_KEY)`.

Reply:
Thank you!